build.gradle

```diff
@@ -18,7 +18,10 @@ allprojects {
 }
 
 ext {
-    compileSdkVersion = 28
-    buildToolsVersion = '28.0.3'
-    androidXLibraryVersion = '1.0.0'
+    compileSdkVersion = 30
```

Comment:
Let's keep compile and target version split. I know that is not something we had before.
But we should have `compileSdkVersion` and `targetSdkVersion` 

That said, when we upgrade to target 30 we need to update the README and the permissions, since some use permissions will be deprecated. Using target 30 will break (like already is) for OS 10 and 11.

My suggestion here is that we upload everything else and leave the target to 28, until we finish the fix for OS 11

Reply:
Hmm do we really need to do that? My other PR will fix the OS 11 issue (and doesn't break it for OS < 10), so if you merge that after this it will work for all OS versions. 
Will update the README.

Reply:
Would be great!

The other PR don't change the external access using `fromFile` and deprecating the old permissions, so it won't fix the whole OS 11 issue, just a part of it

And, even this solution /\  has a know Huawei issue with the path get from the OS because Huawei did it differently. We will need to update more parts of the code before it is OS11 ready

Reply:
Ok, fair point. 
I made the change - target is now back to 28 and the compiled version is 30.